FrameworkSystem/Client/ProxyManagerClient.py

```diff
@@ -147,49 +184,47 @@ def uploadProxy(self, proxy=False, diracGroup=False, chainToConnect=False, restr
       if not result['OK']:
         return S_ERROR("Can't load %s: %s " % (proxyLocation, result['Message']))
 
-    if not chainToConnect:
-      chainToConnect = chain
-
     # Make sure it's valid
-    if chain.hasExpired()['Value']:
+    if chain.hasExpired().get('Value'):
       return S_ERROR("Proxy %s has expired" % proxyLocation)
+    if chain.getDIRACGroup().get('Value') or chain.isVOMS().get('Value'):
```

Comment:
I do not understand this test ? Why is it needed ?

Reply:
In this PR all proxies upload to new table in ProxyDB that store proxies without dirac group and VOMS extensions. When someone try to download proxy group extension will add to "clean" proxy. So this test to check that uploading proxy without extensions.

Reply:
See previous comment: we still want to to be able to upload proxies with DIRAC group